### PR TITLE
Bag sorting

### LINKED ISSUE #6: Wrong item order when importing
When our guild bank is neatly sorted to perfection, the webpage doesn't display it the same way..

Big OCD trigger 😄 

Attached picture of the order it imports the bag into the website.
Think it should be 1-2-3-4 from left to right.

BankBagOrder
WebsiteOrder

### PR DESCRIPTION
**Pull request by codewithbjim**
Rework AddBags function for Importing Encoded Strings to include bagContainerId

### CODE REVIEW

**Comment by codewithbjim** [comment]
Fix for issue #6

**Comment by codewithbjim** [comment]
@athielking

File: ClassicGuildBankData/Migrations/20190609224636_init.cs [hunk 1]
```diff
@@ -252,7 +253,8 @@ protected override void Up(MigrationBuilder migrationBuilder)
                 {
                     SlotId = table.Column<int>(nullable: false),
                     BagId = table.Column<Guid>(nullable: false),
-                    ItemId = table.Column<int>(nullable: true)
+                    ItemId = table.Column<int>(nullable: true),
+                    Notes = table.Column<string>(nullable: true)
```
Comment (athielking) [inline ClassicGuildBankData/Migrations/20190609224636_init.cs]: These columns need to be added as a separate migration

File: ClassicGuildBankData/Migrations/20190609224636_init.cs [hunk 2]
```diff
@@ -227,7 +227,8 @@ protected override void Up(MigrationBuilder migrationBuilder)
                     Id = table.Column<Guid>(nullable: false),
                     ItemId = table.Column<int>(nullable: true),
                     CharacterId = table.Column<Guid>(nullable: false),
-                    isBank = table.Column<bool>(nullable: false)
+                    isBank = table.Column<bool>(nullable: false),
+                    BagContainerId = table.Column<int>(nullable: false)
```
Comment (athielking) [inline ClassicGuildBankData/Migrations/20190609224636_init.cs]: These columns need to be added as a separate migration

**Review by athielking — CHANGES_REQUESTED** [review]
The database schema changes need to be added as a separate migration instead of being added to an existing one.

This is because the DB keeps track of what migrations have already been ran so the migration which these changes were added to has already been ran and is tracked by the database.

Look into the `Add-Migration` command, it should scaffold the changes for you if you've updated the models.

File: ClassicGuildBankData/Migrations/20190609224636_init.cs (hunk 1 above)
Comment (codewithbjim) [reply ClassicGuildBankData/Migrations/20190609224636_init.cs]: @athielking I have pushed with `add-migration` command to migrate model changes

**Review by athielking — APPROVED** [review]
Looks good now.  Migration updated the DB Fine and the import worked as expected.

**Pull request merged**